BetonQuest-core/src/main/java/pl/betoncraft/betonquest/utils/Updater.java

```diff
@@ -86,8 +86,8 @@ public Updater(File file) {
         }
         String version = plugin.getDescription().getVersion();
```

Comment:
This file has a variable URL that needs to be updated.

Reply:
What? can you explain what you mean?

Reply:
This contains a link to Coosh repo. Will propably work for now because its getting redirected but if Coosh decides to fork etc. this will break.
https://github.com/BetonQuest/BetonQuest/blob/b4caaf1e5b6292deda37eaddc8d9cc1e87e8c0ed/BetonQuest-core/src/main/java/pl/betoncraft/betonquest/utils/Updater.java#L46

Reply:
OK, i got it. And there are more problems. I will have a closer look.
This could be also broken
private static final String DEV_DOWNLOAD_LINK = "https://betonquest.pl/{number}/BetonQuest.jar";

Reply:
Solved by future Updater patch